README.md

```diff
@@ -154,25 +161,11 @@ values as needed:
 cp .env.example .env
 ```
 
-You may also need to add basic auth to `app/services/platform.rb`. This
-authentication will be used when talking to platform services such as inventory,
-rbac, and remediations. Anything not deployed locally will require basic auth
-instead of using an identity header (i.e. rbac, remediations):
-
-```diff
-diff --git a/app/services/platform.rb b/app/services/platform.rb
-index 36a0f00..c4b0e92 100644
---- a/app/services/platform.rb
-+++ b/app/services/platform.rb
-@@ -21,6 +21,7 @@ module Platform
-       f.request :retry, RETRY_OPTIONS
-       f.adapter Faraday.default_adapter # this must be the last middleware
-       f.ssl[:verify] = Rails.env.production?
-+      f.basic_auth 'username', 'password'
-     end
-     faraday
-   end
-```
+You may also need to set up basic auth in
+[settings](config/settings/development.yml), option `platform_basic_auth`.
+This authentication will be used when talking to platform services such
+as inventory, rbac, and remediations. Anything not deployed locally will
+require basic auth instead of using an identity header (i.e. rbac, remediations):
 
```

Comment:
Why is this duplicated? This applies for any development setup, so we should place it in the [Development notes](https://github.com/RedHatInsights/compliance-backend/#development-notes) section

Reply:
There isn't a shared prerequisites section, so I duplicated it. I'll figure something out.